### DISCUSSION TITLE
API Key use

### DISCUSSION THREAD

**Discussion by bitner** [supabase/dbdev#196] [Q&A]
I'm creating a python tool to help manage extensions via multiple options / repositories. I was not able to use the token I created when logging into database.dev to access api.database.dev. I am able to use the apikey that is included as part of the examples for installing the dbdev function onto a database, but I am not sure if that key should be used or not. Can you either let me know if it is ok to use that key in another tool or how to go about getting a suitable apikey from a token?

**Comment by imor** [comment]
The token is only needed if you want to publish extensions to database.dev. The apikey included in the examples is sufficient if you want to install extensions.
